libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -1379,13 +1386,13 @@ private static object Divide(object a, object b)
             }
         }
 
-        private static (object value, string error) And(Expression expression, IMemory state)
+        private static (object value, string error) And(Expression expression, IMemory state, Options options)
         {
             object result = true;
             string error = null;
             foreach (var child in expression.Children)
             {
-                (result, error) = child.TryEvaluate(state);
+                (result, error) = child.TryEvaluate(state, new Options());
```

Comment:
>Options [](start = 63, length = 7)

Shouldn't this be initialized from the existing options?  I know right now we only have one property, but seems like the right pattern.

Reply:
replace by a copy constructor.

Reply:
But it is not replaced, right?

---
In reply to: [400615601](https://github.com/microsoft/botbuilder-dotnet/pull/3635#discussion_r400615601) [](ancestors = 400615601)